src/Sarif/Writers/ConsoleLogger.cs

```diff
@@ -202,7 +202,7 @@ private void WriteToConsole(Notification notification)
                     WriteLineToConsole(FormatNotificationMessage(notification, _toolName));
                     break;
                 case FailureLevel.Error:
-                    WriteLineToConsole(FormatNotificationMessage(notification, _toolName), true);
+                    WriteLineToConsole(FormatNotificationMessage(notification, _toolName), forceEmitOfErrorMessages: true);
```

Comment:
>forceEmitOfErrorMessages [](start = 91, length = 24)

this is really 'forceEmitOfErrorNotifications'

Reply:
Done.

---
In reply to: [559828416](https://github.com/microsoft/sarif-sdk/pull/2247#discussion_r559828416) [](ancestors = 559828416)